src/platform/site-wide/sass/merger.scss

```diff
@@ -607,6 +607,10 @@ img[data-srcset] {
   position: relative;
 }
 
+.feature-grey {
+  background-color: #f1f1f1;
+}
+
```

Comment:
We actually have utilities classes for [colors](https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/color.html), to save on some color rules. For this shade of gray, we can use the class `.vads-u-color--gray-lightest`.

Reply:
Sweet!

Reply:
Actually, that class appears to target font color. I did a search for `gray-lightest` and found the color variable, and inserted it for .feature-grey, @ncksllvn

Reply:
Ah, sorry! Looks like there are classes for [background color](https://department-of-veterans-affairs.github.io/vets-design-system-documentation/utilities/background-color.html) as well. Sorry - I'm new to our design system upgrade as well!